### DISCUSSION TITLE
BusROCOF Parameters

### DISCUSSION THREAD

**Discussion by jinningwang** [CURENT/andes#476] [General]
In the graph below, we can observe the effects of BusROCOF's parameters on measurements. Apart from the spikes, the measurements don't exhibit significant differences.

More details can be found here: [BusROCOF.pdf](https://github.com/CURENT/andes/files/12324125/BusROCOF.pdf)

<img src="https://github.com/CURENT/andes/assets/79226045/2812147f-267a-4955-9e4e-71c7c513c0d0" alt="image" width="400">

**Comment by jinningwang** [comment]
@shebuxin

Exercise caution when dealing with spike values while using default parameters, as they have the potential to influence your maximum ROCOF assessment.

**Reply by cuihantao** [reply]
Maybe we should adjust the default parameters for BusFreq?

**Comment by jinningwang** [comment]
I'll do a quick PR to change ``Tw=0.1`` by default.

![image](https://github.com/CURENT/andes/assets/79226045/b05f742a-1b69-4df5-ab78-9f94c48ede75)
